spec/factories/appointments.rb

```diff
@@ -15,8 +15,8 @@
     user
 
     trait :overdue do
-      device_created_at { rand(61..90).days.ago }
-      scheduled_date { rand(35..60).days.ago }
+      device_created_at { rand(81..90).days.ago }
```

Comment:
Did the time range of an overdue appointment change?

Reply:
No, this was something we used to temporarily fix a test. Reverting this.